components/playlist/browser/playlist_constants.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_PLAYLIST_BROWSER_PLAYLIST_CONSTANTS_H_
+#define BRAVE_COMPONENTS_PLAYLIST_BROWSER_PLAYLIST_CONSTANTS_H_
+
+namespace playlist {
+
+extern const char kPlaylistMediaFileUrlKey[];
```

Comment:
I reckon nowadays you can use `constexpr` and keep everything in the header file